Std/Data/Ord.lean

```diff
@@ -0,0 +1,99 @@
+/-
+Copyright (c) 2022 Jannis Limperg. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jannis Limperg
+-/
+
+namespace Ordering
+
+/--
+Check whether the ordering is 'equal'.
+-/
+def isEQ : Ordering → Bool
```

Comment:
```suggestion
def isEq : Ordering → Bool
```

Reply:
Done, also for the other functions in this namespace.

Reply:
Oh, I just realised: the reason it was `isEQ` in the first place is that core has `isLE`. So this introduces an inconsistency.

Reply:
Oh, I just realised: the reason it was `isEQ` is that core has `isLE`. So this introduces an inconsistency.

Reply:
The suggestion is just for `Eq`, not `LE`. We have discussed this naming "inconsistency" in the mathlib4 stream and decided that it's not inconsistent, because `LE` is an acronym for less-equal and `Eq` is an abbreviation. (Just don't think too hard about `Ne`...)

Reply:
Okay, I've changed `isGe` etc. back to `isGE`. I've also added `isNe`.